persistence/sql/persister_identity.go

```diff
@@ -257,9 +259,25 @@ func (p *Persister) normalizeAllAddressess(ctx context.Context, id *identity.Ide
 
 func (p *Persister) normalizeVerifiableAddresses(ctx context.Context, id *identity.Identity) {
 	for k := range id.VerifiableAddresses {
-		id.VerifiableAddresses[k].IdentityID = id.ID
-		id.VerifiableAddresses[k].NID = p.NetworkID(ctx)
-		id.VerifiableAddresses[k].Value = stringToLowerTrim(id.VerifiableAddresses[k].Value)
+		v := id.VerifiableAddresses[k]
+
+		v.IdentityID = id.ID
+		v.NID = p.NetworkID(ctx)
+		v.Value = stringToLowerTrim(v.Value)
+		v.Via = x.Coalesce(v.Via, identity.AddressTypeEmail)
+		if len(v.Status) == 0 {
+			if v.Verified {
+				v.Status = identity.VerifiableAddressStatusCompleted
+			} else {
+				v.Status = identity.VerifiableAddressStatusPending
+			}
+		}
+
+		if v.Verified && (v.VerifiedAt == nil || time.Time(*v.VerifiedAt).IsZero()) {
```

Comment:
Does this mean, you can set the email address to `verified` in the webhook but not set the timestamp? Shouldn't that be possible as well? 

And could you add some comments around this to explain what's going on in this (changed) part? I find it a bit difficult to gauge what this is for.

Reply:
You can do both - if you set verified to true, the timestamp is now. If you set both, you control both values.

Reply:
```suggestion
		// If verified is true but no timestamp is set, we default to time.Now
		if v.Verified && (v.VerifiedAt == nil || time.Time(*v.VerifiedAt).IsZero()) {
```